data/json/items/tools.json

```diff
@@ -103,7 +103,31 @@
             "moves_cost": 15
     }
   },
-
+  {
+   "id":"crude_firestarter",
+   "type": "TOOL",
+   "symbol": ",",
+   "color": "light_gray",
+   "name": "electric firestarter",
+   "description": "This is a crudely made electric firestarter, which can function as an inefficient lighter.",
+   "price": 100,
+   "material": "steel",
+   "weight": 14,
+   "volume": 1,
+   "bashing": 0,
+   "cutting": 0,
+   "to_hit": 0,
+   "max_charges": 100,
+   "initial_charges": 0,
+   "charges_per_use": 5,
+   "turns_per_charge": 0,
+   "ammo": "battery",
+   "revert_to": "null",
+   "use_action": {
+            "type": "firestarter",
+            "moves_cost": 15
```

Comment:
That moves cost is really low for something that probably takes a while to start a fire. With 5 battery/use and no amplifier, it probably has a rather low current. It should probably even use the extended firestarter rather than the quick one.